core/server/master/src/main/java/alluxio/master/metastore/BlockStore.java

```diff
@@ -83,6 +83,13 @@
    */
   void close();
 
+  /**
+   * Get the number of blocks in the block store.
+   *
+   * @return size of the block store
+   */
```

Comment:
nit we can simplify the javadoc here

```suggestion
  /**
   * @return size of the block store
   */
```